<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana</class>
 <widget class="QWidget" name="Ventana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="Boton" name="botonDentista" native="true"/>
   </item>
   <item row="4" column="0">
    <widget class="Boton" name="botonBoca" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="Boton" name="botonHigiene" native="true"/>
   </item>
   <item row="5" column="0">
    <widget class="Boton" name="botonCepillar" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="Boton" name="botonVisitas" native="true"/>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Boton</class>
   <extends>QWidget</extends>
   <header>boton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
